src/gordon_gcp/clients/http.py

```diff
@@ -38,21 +34,20 @@
 """
 
 import datetime
-import http.client
 import json
 import logging
+from functools import partial
+from http import HTTPStatus
```

Comment:
These two imports goes against [our following](https://github.com/spotify/gordon-gcp/blob/develop/CONTRIBUTING.rst#code) of Google's style guide.

Reply:
Will fix.